src/test/java/CommandReminderTest.java

```diff
@@ -0,0 +1,28 @@
+import duke.command.CommandReminder;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.TaskType;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class CommandReminderTest {
+
+    @Test
+    void execute() {
+        TaskList taskList = new TaskList();
+        Task testTask = TaskList.createTask(TaskType.EVENT,"something/by somewhen");
+        taskList.addTask(testTask);
+        CommandReminder testCommand = new CommandReminder();
+        testCommand.execute(taskList);
+
+        Task mainTask = taskList.getList().get(0);
+        assertEquals(true, mainTask.isQueuedTasks());
+//        Task queuedTask = mainTask.getQueuedTasks().getList().get(0);
```

Comment:
I assume this is where you want me to comment. What I did here was to create a Task with some queued Tasks assigned to it. So _mainTask_ is a Task Object that has a TaskList Object called _queuedTasks_. 

As we know, the TaskList Object is not an ArrayList by itself so I call **.getList** to get the ArrayList and I want to compare the first element in that array (the first queued Task) so I call **.get(0)**